docs/cristin.ttl

```diff
@@ -1243,12 +1247,65 @@ It is a self reference.""" ;
     rdfs:domain  :VARBEID_HRCS_KLASSIFISERING_Class ;
     :isignored   "true" .
 
-
 :aktivitetskode
     a            :cristinfield ;
     rdfs:comment """Research Activity Code in the UKCRC Health Research Classification System (HRCS).
                      See https://hrcsonline.net/research-activities/.
                      """ ;
     rdfs:range   xsd:string ;
     rdfs:domain  :VARBEID_HRCS_KLASSIFISERING_Class ;
-    :isignored   "true" .
\ No newline at end of file
+    :isignored   "true" .
+
+:TYPE_BOK_RAPPORT_DEL
+    a            :cristinfield ;
+    rdfs:comment """Reference to a Cristin entry that is a part of a book or report.""" ;
+    rdfs:range   :TYPE_BOK_RAPPORT_Class ;
+    rdfs:domain  :Result .
+
+:varbeidlopenr_inngar_i
+    a            :cristinfield ;
+    rdfs:comment """The CristinID of the current TYPE_BOK_RAPPORT_DEL_Class entry's parent TYPE_BOK_RAPPORT_Class entry.""" ;
+    rdfs:domain  :TYPE_BOK_RAPPORT_DEL_Class ;
+    rdfs:range   xsd:integer .
+
+:status_utgitt_av_forlag
+    a            :cristinfield ;
+    rdfs:comment """UNKNOWN meaning. It takes values "J" and "N" corresponding to the values true and false.""" ;
```

Comment:
We are investigating what the meaning of that field is in this context. Will update accordingly.